docs/advanced_documentation/build-guide.md

```diff
@@ -160,10 +160,11 @@ WSL), or in a physical/virtual machine.
 Append the following lines into the file `${HOME}/.bashrc`.
 
 ```shell
-export CXX=clang++-14  # or g++-11
-export CC=clang-14  # gcc-11
+export CXX=clang++-14            # or g++-11
+export CC=clang-14               # gcc-11
 export CMAKE_PREFIX_PATH=/home/linuxbrew/.linuxbrew
 export LLVM_COV=llvm-cov-14
+export CLANG_TIDY=clang-tidy-14  # only if you want to use one of the clang-tidy presets
```

Comment:
Just make it mandatory in the build guide. We just use it always in the presets.